utils/grpchelpers/blockgrpcconn.go

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2024 Renesas Electronics Corporation.
+// Copyright (C) 2024 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grpchelpers
+
+import (
+	"context"
+	"sync"
+
+	"github.com/aosedge/aos_common/aoserrors"
+	"google.golang.org/grpc"
+)
+
+// BlockingGRPCConn allows to Set/Stop connection multiple times. All incoming requests are blocked for stopped
+// connection until it is reset.
+type BlockingGRPCConn struct {
```

Comment:
Rename to just `GRPCConnection`. As you don't have async connection implementation to distinguish with.

Reply:
done